### PR TITLE
Added group for about info endpoint

### PR DESCRIPTION
**Pull request by ssd04**
- add endpoint for proxy version
- manage app version and commit from build ldflags
- create separate about info processor for future metrics in about section

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
 	go clean -testcache ./...
 
 test: clean-test
 	go test ./...
+
+build:
+	cd ${cmd_dir} && \
+		go build -v \
+		-o ${binary}
+	-ldflags="-X main.appVersion=$(shell git describe --tags --long --dirty) -X main.commitID=$(shell git rev-parse HEAD)"
```
Comment (iulianpascalau) [inline Makefile:30]: :+1:

File: process/aboutInfoProcessor.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+	return &aboutProcessor{
+		commitID:   commit,
+		appVersion: appVersion,
+	}, nil
+}
+
+func (ap *aboutProcessor) GetAboutInfo() *data.GenericAPIResponse {
+	commit := ap.commitID
+	if ap.commitID != common.UndefinedCommitString {
+		commit = commit[:7]
```
Comment (iulianpascalau) [inline process/aboutInfoProcessor.go:31]: magic number. Define a const?

Comment (ssd04) [reply process/aboutInfoProcessor.go]: done, added const

File: cmd/proxy/config/apiConfig/v1_0.toml [hunk 3]
```diff
@@ -9,6 +9,11 @@
 # RateLimit: if set to 0, then the endpoint won't be limited. Otherwise, a given IP address can only make a number of
 # requests in a given time stamp, configurable in config.toml
 
+[APIPackages.about]
+Routes = [
+    { Name = "", Open = true, Secured = true, RateLimit = 0 },
```
Comment (bogdan-rosianu) [inline cmd/proxy/config/apiConfig/v1_0.toml:14]: `Secured = false` => this can be a public info

File: cmd/proxy/config/apiConfig/v_next.toml [hunk 4]
```diff
@@ -9,6 +9,11 @@
 # RateLimit: if set to 0, then the endpoint won't be limited. Otherwise, a given IP address can only make a number of
 # requests in a given time stamp, configurable in config.toml
 
+[APIPackages.about]
+Routes = [
+    { Name = "", Open = true, Secured = true, RateLimit = 0 },
```
Comment (bogdan-rosianu) [inline cmd/proxy/config/apiConfig/v_next.toml:14]: `Secured = false` => this can be a public info

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: cmd/proxy/config/apiConfig/v1_0.toml (hunk 3 above)
Comment (ssd04) [reply cmd/proxy/config/apiConfig/v1_0.toml]: done

File: cmd/proxy/config/apiConfig/v_next.toml (hunk 4 above)
Comment (ssd04) [reply cmd/proxy/config/apiConfig/v_next.toml]: done

**Review by iulianpascalau — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by tanghel — APPROVED** [review]

**Review by gabi-vuls — APPROVED** [review]
System test passed.

**Pull request merged**